matlab/MeCorr/MATLAB_CODE_allrxns.m

```diff
@@ -0,0 +1,419 @@
+ctd2compoundidname_id.Properties.VariableNames{'ctd2compoundidname_id'}= 'cpd_id';
+celllinenames_ccle1= array2table(celllinenames_ccle1);
+celllinenames_ccle1.Properties.VariableNames{'celllinenames_ccle1'}= 'ccl_name';
+drug_auc_expt= array2table(drug_auc_expt);
+drug_auc_expt.Properties.VariableNames{'drug_auc_expt3'}= 'cpd_id';
+drug_auc_expt.Properties.VariableNames{'drug_auc_expt1'}= 'experiment_id';
+drug_auc_expt.Properties.VariableNames{'drug_auc_expt2'}= 'auc';
+exptidcelllinemediamatch=exptidcelllinemediamatch(2:end, :);
+exptidcelllinemediamatch= array2table(exptidcelllinemediamatch); %s4
+exptidcelllinemediamatch.Properties.VariableNames{'exptidcelllinemediamatch1'}= 'experiment_id';
+exptidcelllinemediamatch.Properties.VariableNames{'exptidcelllinemediamatch2'}= 'master_ccl_id';
+hcommon1= array2table(hcommon1);
+hcommon1.Properties.VariableNames{'hcommon1'}= 'experiment_id';
+
+MODE = 1;  % reaction (1) or gene list (0)
+epsilon = 1E-2; rho = 1;
+kappa = 1; % parameters for integrating transcriptomics data. kappa is the strength of down regulation of genes (Chandrasekaran & Price, PNAS, 2010)
+hmetransint = 1;
+basalflag = 1;
+
+hmei_list = {'BRD-A02303741';'BIX-01294';'methylstat';'QW-BI-011';...
+    'UNC0321';'CBB-1007';'UNC0638';'GSK-J4'};
+hmei_list= cell2table(hmei_list);
+hmei_list.Properties.VariableNames{'hmei_list'}='cpd_name';
+cpd_info= readtable('./../DataRees2016/nchembio.1986-S3_cpd.xlsx');
+cpd_infoMe= cpd_info(ismember(cpd_info(:, 1), hmei_list), [1,5,6,7,8]);
+cpd_infoMe= table2cell(cpd_infoMe);
+
+% If I don't want to run next for-loop:
+%load('VariablesSaved\fluxstate_gurobi');
+%load('VariablesSaved\grate_ccle_exp_soft');
+%% long for-loops
+weights= [1E-1, 1E-2, 1E-3, 1E-4, 1E-5, 1E-6];
+name= [1, 2, 3, 4, 5, 6];
+for i_weight = 2:length(weights)
+    epsilon_methylation= weights(i_weight);
+    grate_ccle_exp_soft = NaN(height(exptidcelllinemediamatch),3); %contains acetylation flux based on basal metabolic state
+    fluxes_allrxns= NaN(height(exptidcelllinemediamatch),length(model2.rxns));
+    g_rate= NaN(height(exptidcelllinemediamatch), length(model2.rxns));
+    
+    for i = 1:height(exptidcelllinemediamatch)
+        % match cell line data in CCLE with CTD2
+        ii = find(ismember(ctd2celllineidname_id, exptidcelllinemediamatch(i,2)));
+        iii = find(ismember(celllinenames_ccle1, ctd2celllineidname(ii,1)));
+        if ~isempty(iii)
+            iii  = iii(1);
+            %        model2 = min_model;
+            %        model2 = metabolicmodel;
+            
+            %find up and down-regulated genes in each cell line
+            ongenes = unique(ccleids_met(ccle_expression_metz(:,iii) > 2));
+            offgenes = unique(ccleids_met(ccle_expression_metz(:,iii) < -2));
+            
+            %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+            %  set the glucose uptake based on media

```

Comment:
These lower bounds only change glucose abundance for 4 medium conditions. The code `matlab/scripts/nutrient_sensitivty/medium_LB_constraints.m` in `master` can improve your drug sensitivity predictions.